app/code/Magento/Backend/Block/Media/Uploader.php

```diff
@@ -28,15 +30,24 @@ class Uploader extends \Magento\Backend\Block\Widget
     protected $_fileSizeService;
 
     /**
+     * @var \Magento\Framework\Json\EncoderInterface
+     */
+    private $jsonEncoder;
+
+    /**
      * @param \Magento\Backend\Block\Template\Context $context
      * @param \Magento\Framework\File\Size $fileSize
      * @param array $data
+     * @param \Magento\Framework\Json\EncoderInterface $jsonEncoder
      */
     public function __construct(
         \Magento\Backend\Block\Template\Context $context,
         \Magento\Framework\File\Size $fileSize,
-        array $data = []
+        array $data = [],
+        \Magento\Framework\Json\EncoderInterface $jsonEncoder = null
     ) {
+        $this->jsonEncoder = $jsonEncoder ?:
+            ObjectManager::getInstance()->get(\Magento\Framework\Json\EncoderInterface::class);
```

Comment:
Try using imports (Alt+Enter in PhpStorm) at least for such cases, probably it could fit in one line.